test/SlowTests/Issues/RavenDB-19481.cs

```diff
@@ -0,0 +1,157 @@
+﻿using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using FastTests;
+using FastTests.Graph;
```

Comment:
remove this, since it doesn't exists in 6.0

Reply:
Done